nomad.sysext/create.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# vim: et ts=2 syn=bash
+#
+
+RELOAD_SERVICES_ON_MERGE="true"
+
+# Fetch and print a list of available versions.
+# Called by 'bakery.sh list <sysext>.
```

Comment:
Can be removed, this is a left-over from the skeleton script instructing implementers what to do.